app/views/admin/badges/edit.html.erb

```diff
@@ -14,8 +14,8 @@
     </div>
 
     <div class="form-group">
-      <%= form.label :slug, "Slug:" %>
-      <%= form.text_field :slug, class: "form-control" %>
+      <%= form.label :slug, "Slug - auto generated based on the title." %>
+      <%= form.text_field :slug, class: "form-control", disabled: true %>
```

Comment:
I suppose we could remove it altogether like I did below in `new.html.erb`. I wanted to keep it so that admins know what's going on here.